Index: xen-4.4.0~rc3+20140210/tools/firmware/Makefile
===================================================================
--- xen-4.4.0~rc3+20140210.orig/tools/firmware/Makefile	2014-02-10 12:56:18.042749731 +0100
+++ xen-4.4.0~rc3+20140210/tools/firmware/Makefile	2014-02-10 12:56:18.614752527 +0100
@@ -9,9 +9,6 @@ SUBDIRS-y :=
 ifeq ($(OVMF_PATH),)
 SUBDIRS-$(CONFIG_OVMF) += ovmf-dir
 endif
-ifeq ($(SEABIOS_PATH),)
-SUBDIRS-$(CONFIG_SEABIOS) += seabios-dir
-endif
 SUBDIRS-$(CONFIG_ROMBIOS) += rombios
 SUBDIRS-$(CONFIG_ROMBIOS) += vgabios
 SUBDIRS-y += hvmloader
@@ -52,9 +49,6 @@ distclean: subdirs-distclean
 subdir-distclean-ovmf: .phony
 	rm -rf ovmf-dir ovmf-dir-remote
 
-subdir-distclean-seabios-dir: .phony
-	rm -rf seabios-dir seabios-dir-remote
-
 .PHONY: ovmf-dir-force-update
 ovmf-dir-force-update:
 	set -ex; \
Index: xen-4.4.0~rc3+20140210/tools/firmware/hvmloader/Makefile
===================================================================
--- xen-4.4.0~rc3+20140210.orig/tools/firmware/hvmloader/Makefile	2014-02-10 12:56:18.042749731 +0100
+++ xen-4.4.0~rc3+20140210/tools/firmware/hvmloader/Makefile	2014-02-10 12:56:18.614752527 +0100
@@ -39,7 +39,7 @@ CIRRUSVGA_DEBUG ?= n
 
 OVMF_DIR := ../ovmf-dir
 ROMBIOS_DIR := ../rombios
-SEABIOS_DIR := ../seabios-dir
+SEABIOS_DIR := /usr/share/seabios
 
 ifeq ($(CONFIG_ROMBIOS),y)
 STDVGA_ROM    := ../vgabios/VGABIOS-lgpl-latest.bin
@@ -74,11 +74,7 @@ endif
 ifeq ($(CONFIG_SEABIOS),y)
 OBJS += seabios.o
 CFLAGS += -DENABLE_SEABIOS
-ifeq ($(SEABIOS_PATH),)
-	SEABIOS_ROM := $(SEABIOS_DIR)/out/bios.bin
-else
-	SEABIOS_ROM := $(SEABIOS_PATH)
-endif
+SEABIOS_ROM := $(SEABIOS_DIR)/bios.bin
 ROMS += $(SEABIOS_ROM)
 endif
 
