Description: Export flags from configure
Author: Bastian Blank <waldi@debian.org>
Origin: vendor
Forwarded: not-needed
--- a/config/Tools.mk.in
+++ b/config/Tools.mk.in
@@ -26,6 +26,9 @@
 SEABIOS_PATH        := @seabios_path@
 OVMF_PATH           := @ovmf_path@
 
+CFLAGS              += @CFLAGS@
+CPPFLAGS            += @CPPFLAGS@
+
 # Extra folder for libs/includes
 PREPEND_INCLUDES    := @PREPEND_INCLUDES@
 PREPEND_LIB         := @PREPEND_LIB@
